src/components/Categories.jsx

```diff
@@ -0,0 +1,21 @@
+import { useDispatch } from 'react-redux';
+import { checkStatus } from '../redux/categories/categories';
+
+const Categories = () => {
+  const dispatch = useDispatch();
+  return (
+    <div className="categories">
+      <h1>Categories list</h1>
+      <button
+        type="button"
+        onClick={() => {
+          dispatch(checkStatus());
+        }}
+      >
+        Check Status
+      </button>
```

Comment:
- [ ] Good job on dispatching `checkStatus` action but Please make sure that when the user clicks the `Check Status` button, the text `Under Construction` appear on the page.

Reply:
I have implemented this functionality as requested.